ios/browser/ui/webui/DEPS

```diff
@@ -1,6 +1,22 @@
 include_rules = [
   "+brave/components/brave_account/features.h",
+  "+brave/components/brave_private_cdn",
```

Comment:
I don't see `include`s in this PR using this, so am I right to assume they've been missing for already existing code?
Same for `components/image_fetcher/ios`, `google_apis/gaia`, `ios/chrome/browser/signin/model`, and `third_party/skia/include/encode`.

Reply:
`sanitized_image_source.mm` is using `#include "brave/components/brave_private_cdn/private_cdn_helper.h"`
`sanitized_image_source.mm:25:11`: `#include "components/image_fetcher/ios/ios_image_decoder_impl.h"`

The rest are used the same way.